
PLAY [autohdmap_multilane_0] ***************************************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************************************************************************************************************
ok: [10.11.5.137]

TASK [autohdmap_multilane_0 : run container for autohdmap_multilane_0 op-01.gzproduction.com/autohdmap/autohdmap_multilane:v1.7.1] *********************************************************************************************************************************************
changed: [10.11.5.137]

TASK [autohdmap_multilane_0 : make autohdmap config] ***************************************************************************************************************************************************************************************************************************
 [ERROR]: User interrupted execution
Process WorkerProcess-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/local/lib/python2.7/dist-packages/ansible/executor/process/worker.py", line 118, in run
    self._final_q
  File "/usr/local/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 139, in run
    res = self._execute()
  File "/usr/local/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 603, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/template.py", line 203, in run
    result.update(copy_action.run(task_vars=task_vars))
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/copy.py", line 510, in run
    module_return = self._copy_file(source_full, source_rel, content, content_tempfile, dest, task_vars, follow)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/copy.py", line 258, in _copy_file
    dest_status = self._execute_remote_stat(dest_file, all_vars=task_vars, follow=follow, checksum=force)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/__init__.py", line 517, in _execute_remote_stat
    wrap_async=False)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/__init__.py", line 839, in _execute_module
    res = self._low_level_execute_command(cmd, sudoable=sudoable, in_data=in_data)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/action/__init__.py", line 954, in _low_level_execute_command
    rc, stdout, stderr = self._connection.exec_command(cmd, in_data=in_data, sudoable=sudoable)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/connection/ssh.py", line 1081, in exec_command
    (returncode, stdout, stderr) = self._run(cmd, in_data, sudoable=sudoable)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/connection/ssh.py", line 337, in wrapped
    return_tuple = func(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/connection/ssh.py", line 971, in _run
    return self._bare_run(cmd, in_data, sudoable=sudoable, checkrc=checkrc)
  File "/usr/local/lib/python2.7/dist-packages/ansible/plugins/connection/ssh.py", line 808, in _bare_run
    events = selector.select(timeout)
  File "/usr/local/lib/python2.7/dist-packages/ansible/compat/selectors/_selectors2.py", line 466, in select
    maxevents=max_events)
  File "/usr/local/lib/python2.7/dist-packages/ansible/compat/selectors/_selectors2.py", line 119, in _syscall_wrapper
    result = func(*args, **kwargs)
KeyboardInterrupt
[Finish] Failed to deploy [op-01.gzproduction.com/autohdmap/autohdmap_multilane:v1.7.1]